test/unit/feature_engineering/test_general.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+from unittest import TestCase
+
+import pandas as pd
+import numpy as np
+
+from openstf.feature_engineering.general import enforce_feature_order
+
+
+class TestGeneral(TestCase):
+    def test_enforce_feature_order_with_horizon_columns(self):
+
+        df = pd.DataFrame(
+            np.arange(12).reshape(3, 4), columns=["load", "Horizon", "A", "E"]
+        )
+
+        result = enforce_feature_order(df)
+
+        self.assertEqual(result.columns.to_list(), ["load", "A", "E", "Horizon"])
```

Comment:
Minor suggestion. if you use a variable for the columns you could test against a sorted version. This makes it even more clear that you expect the columns to be sorted:
```python
columns = ["load", "Horizon", "A", "E"]
self.assertEqual(result.columns.to_list(), sorted(columns))
```